src/Features/ERDDAP/Platform/Forecasts/Page/index.tsx

```diff
@@ -88,18 +87,24 @@ export const Forecast = ({ platform, forecast_type, ...props }: Props) => {
     })
   }
 
-  forecastResults.forEach(({ result, meta }, index) => {
-    if (result?.data) {
-      chartData.push({
-        timeSeries: result.data as ReadingTimeSeries[],
-        name: meta.name + " - forecast",
-        unit: meta.units,
-        url: meta.source_url,
-        dashStyle: "Solid",
-        color: colorCycle[index + 1],
-      })
-    }
-  })
+  forecastResults
+    .filter((r) => {
+      if (r.result.data) {
+        return r.result.data[0].time > threeDaysAgoRounded()
```

Comment:
I think we might want to check the most recent time, rather than the first time in the forecast. 

FVCOM is kinda screwy in that it's a five day 'forecast' but it's started 2 days back. So the earliest day can still 4 days ago, but there is still a day that's ahead.

Reply:
Just coming back to this given the issue in buoy barn this morning. 

I'm checking the first time/index in the forecast/array so I don't include the forecasts from Oct (iterating through and not using any that are more than 3 days ago). Given that info about FVCOM having the earliest start day be max 4 days ago, it seems like I should still check the first time in the forecast but instead of `threeDaysAgo` it should be at least four days ago, so include any forecast data that is within the last four days.

I'm not 100% sure what we'd be checking if we were to use the most recent time in the forecast/last index in the array as those are the dates are the same in both forecasts models.